.scalafmt.conf

```diff
@@ -21,3 +21,8 @@ align.openParenDefnSite = false
 danglingParentheses = true
 
 trailingCommas = preserve
+
+# TODO update scalafmt and enable Scala 3
```

Comment:
scalafmt 2.x does not support Scala 3.
https://github.com/sbt/sbt/blob/5c90a84c72ca6aae1f08dae1cd1467da6a3d29fd/.scalafmt.conf#L1-L2